coop_env.py

```diff
@@ -88,7 +89,7 @@ def _get_total_reward(self):
     def _step(self, actions):
         """Execute the specified list of actions.
         """
-        assert self.action_space.contains(actions)
+        # assert self.action_space.contains(actions)
```

Comment:
Correct me if I'm wrong, but the max_accel is set to a default of 1, and I don't see anything which doesn't use this default.